### ISSUE TITLE
v19.35.3-3.0.4 Can’t select hdr quality with ytclassicvideoquality

### ISSUE THREAD

**Issue by ksagameng2** [qnblackcat/uYouPlus#1564]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you read the FAQ?

- [X] Yes, I have read the FAQ

### Device information

- iOS/iPadOS version: 17.0
- Device model: iPhone 12
- Sideload tool (AltStore, Sideloadly, TrollStore, etc.): Trollstore

### IPA source

Armconverter

### Describe the issue

You cannot select hdr video quality with the ytclassicvideoquality tweak enabled

### Steps to reproduce

1. Open an hdr video
2. Open the quality selector
3. No hdr options

### Are you using the latest version of uYouPlus? If not, why?

✅ Yes, I'm using the latest version of uYouPlus

### Does the issue happen with the official YouTube app from the App Store?

❌ No, I cannot reproduce this issue in the official YouTube app from the App Store

### Additional information

Seems to be related to the fact that when youtube used this selector there was no support for hdr video on ios yet

**Comment by PoomSmart** [comment]
I don't think this is related to YTClassicVideoQuality as on my device they show just fine.
